ivy/functional/frontends/torch/creation_ops.py

```diff
@@ -202,3 +202,24 @@ def as_tensor(
     device=None,
 ):
     return ivy.asarray(data, dtype=dtype, device=device)
+
+
+@to_ivy_arrays_and_back
+def zeros_like_v_0p3p0_to_0p3p1(input, out=None):
```

Comment:
sorry for the dealy, but can you tell me why you have this function names instead of the expected zeros_like. Thank you!